gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/diagnosticreport/SpecimenCompoundsMapper.java

```diff
@@ -189,6 +193,10 @@ private void handleNarrativeStatements(RCMRMT030101UKCompoundStatement compoundS
                 compoundStatement.getConfidentialityCode(),
                 childNarrativeStatement.getConfidentialityCode());
 
+            if (observation != null && observation.getMeta().hasSecurity()) {
```

Comment:
I think it will be good to unify the condition with line 162 so that they look the same. I think here we can get a nullPointer exception as we don't check whether "observation.getMeta()" itself is null.